javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/reflectionmodel/ReflectionClassDeclaration.java

```diff
@@ -154,6 +154,10 @@ public String getQualifiedName() {
                 methods.add(ref.getCorrespondingDeclaration());
             }
         }
+        // when empty there is no sense in trying to find the most applicable
```

Comment:
I would add also the comment that it helps debugging, because the performance gain should be negligible